cmd/minikube/cmd/service.go

```diff
@@ -72,20 +72,24 @@ var serviceCmd = &cobra.Command{
 			cmdutil.MaybeReportErrorAndExit(err)
 		}
 
-		url, err := cluster.GetServiceURL(api, namespace, service, serviceURLTemplate)
+		urls, err := cluster.GetServiceURLs(api, namespace, service, serviceURLTemplate)
 		if err != nil {
 			fmt.Fprintln(os.Stderr, err)
 			fmt.Fprintln(os.Stderr, "Check that minikube is running and that you have specified the correct namespace (-n flag).")
 			cmdutil.MaybeReportErrorAndExit(err)
 		}
-		if https {
-			url = strings.Replace(url, "http", "https", 1)
-		}
-		if serviceURLMode {
-			fmt.Fprintln(os.Stdout, url)
-		} else {
-			fmt.Fprintln(os.Stdout, "Opening kubernetes service "+namespace+"/"+service+" in default browser...")
-			browser.OpenURL(url)
+		for _, url := range urls {
+			if https {
+				url = strings.Replace(url, "http", "https", 1)
+			}
+			if serviceURLMode {
+				fmt.Fprintln(os.Stdout, url)
+			} else {
+				if strings.HasPrefix(url, "http") {
```

Comment:
If it doesn't have http prefix it looks like it's just going to be ignored? Perhaps print it out at least?